Spec/Auth.swift

```diff
@@ -1106,6 +1106,36 @@ class Auth : QuickSpec {
                         done()
                     }
                 }
+                
+                tokenParams.clientId = "newClientId"
+                tokenParams.ttl = 2000
+                tokenParams.capability = "{ \"test:*\":[\"test\"] }"
+                
+                waitUntil(timeout: testTimeout) { done in
+                    rest.auth.createTokenRequest(tokenParams, options: authOptions) { tokenRequest, error in
```

Comment:
This API does not look right.  Why is this `options:` when the IDL specifies we have two args, `TokenParams` and `AuthOptions`.  Whilst I appreciate this PR is not necessarily about this, it's important we fix things as we go along.  Can you take a look please?